### DISCUSSION TITLE
Edit configuration file....

### DISCUSSION THREAD

**Discussion by pepsonEL** [john30/ebusd#1167] [General]
Hi. Can anybody tell me step by step how change and use custom configuration? How edit file etc... i can not find any manuall on github.

**Comment by Koky05** [comment]
1. Create clon of configuration files from git on local
2. Copy configuration for HA autodiscovery [MQTT Integration](https://github.com/john30/ebusd/wiki/MQTT-integration)
3. Start eBus deamon with parameters: `--configpath=/etc/ebusd/ebusd-2.1.x/en` (your local address) [Message configuration options](https://github.com/john30/ebusd/wiki/2.-Run#message-configuration-options)
      `--mqttint=/etc/ebusd/mqtt-hassio.cfg`
5. Change what you want with your message decoding [Message definition](Message definition)
6. After every change reload configuration `ebusctl reload` [TCP client commands](https://github.com/john30/ebusd/wiki/3.1.-TCP-client-commands)

**Reply by pepsonEL** [reply]
but i dont know how change config file....
and what you mean of point 2 ?

**Reply by Koky05** [reply]
Configuration files is basic csv text file. You can edit it in every text editor or open it with Excel.
Default configuration file for HA Autodiscovery `mqtt-hassio.cfg` contains filters for read-only messages and only for some subset of messages: `filter-name = status|temp|humidity|yield|count|energy|power|runtime|hours|starts|mode|curve|^load$|^party$|sensor|timer`
So if you use HA and want to see every message you need make it local and also change some configuration.

**Reply by pepsonEL** [reply]
But how change configuration ? From read to write ?
I use addon ebusd in my HA. But when use addon where i can find this cfg file ? I need edit this cfg ? And where replace it to use in addon this my edited.

**Reply by Koky05** [reply]
Try to look into HA-Addon [documentation](https://github.com/LukasGrebe/ha-addons/blob/main/ebusd/DOCS.md#how-to-run-ebusd)
